workbench/tests/renderer/app.test.js

```diff
@@ -89,7 +84,7 @@ describe('Various ways to open and close InVEST models', () => {
     expect(executeButton).toBeDisabled();
     const setupTab = await findByText('Setup');
     expect(setupTab.classList.contains('active')).toBeTruthy();
-    expect(getSpec).toHaveBeenCalledTimes(1);
+    expect(getSpec).toHaveBeenCalledTimes(2);
```

Comment:
good question, it's because `DataDownloadModal` now calls `getSpec` to get the sampledata info for each model. In the test environment we've mocked so there is only one model. So we get one `getSpec` call when rendering the app, and then the second when we click to open a model.